public/index.html

```diff
@@ -8,6 +8,7 @@
 
     <!-- Lato regular, regular italic -->
     <link href="https://fonts.googleapis.com/css2?family=Lato:ital@0;1&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="https://unpkg.com/bulma-modal-fx/dist/css/modal-fx.min.css" />
```

Comment:
What does this do? I took a quick look and it seems like `bulma-modal-fx`'s effects can be achieved with [`animate.css`](https://daneden.github.io/animate.css/), which we already depend on - our third-party CSS bundle is already quite large, let's try to avoid adding additional dependencies unless absolutely necessary